cmake/public/LoadHIP.cmake

```diff
@@ -126,6 +133,11 @@ IF(HIP_FOUND)
   FIND_LIBRARY(hipblas_LIBRARIES hipblas HINTS ${HIPBLAS_PATH}/lib)
   FIND_LIBRARY(hipsparse_LIBRARIES hipsparse HINTS ${HIPSPARSE_PATH}/lib)
 
+
+  # Necessary includes for building PyTorch since we include HIP headers that depend on hcc/hsa headers.
+  set(hsa_INCLUDE_DIRS ${HCC_PATH}/include)
```

Comment:
can this be found via `find_package(hsa REQUIRED)` and let their cmake files populate the has_INCLUDE_DIRS?
line 109 - 119